exercises/anagram/src/anagram.h

```diff
@@ -1,13 +1,33 @@
 #ifndef ANAGRAM_H
 #define ANAGRAM_H
 
-#define MAX_STR_LEN 20
+enum anagramStatus {
+   UNCHECKED = -1,
+   NOT_ANAGRAM,
+   IS_ANAGRAM
+};
 
-struct vector {
-   char (*vec)[MAX_STR_LEN];
-   int size;
+struct candidate {
+   enum anagramStatus isAnagram;
+   char *candidate;
 };
 
-struct vector anagrams_for(char *, struct vector);
+struct candidates {
+   struct candidate *candidate;
+   int size;
```

Comment:
I find that `count` is usually more clear because `size` could be in bytes or in number of elements.

Reply:
I'd also go with `size_t` or some other unsigned type since you probably aren't going to want to specify that you provided -4 candidates.